exonum/src/storage/proof_map_index/key.rs

```diff
@@ -259,52 +259,93 @@ impl PartialEq for DBKey {
 
 impl ::std::fmt::Debug for DBKey {
```

Comment:
IMO, this implementation could be tested; if not for anything else, then for simply showing how output is intended to look in a [TDD](https://en.wikipedia.org/wiki/Test-driven_development) fashion. I'm not sure whether this aligns with best practices for Rust coding.

Reply:
Ready